task1/lib/main.dart

```diff
@@ -0,0 +1,157 @@
+import 'package:flutter/material.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Task 1',
+      home: MainPage(),
+    );
+  }
+}
+
+class MainPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(centerTitle: true, title: Text('Flutter Vadodara')),
+      body: Container(
+        padding: EdgeInsets.only(top: 16, bottom: 16),
+        margin: EdgeInsets.fromLTRB(32, 0, 32, 0),
+        child: Column(
+          children: [
+            Container(
```

Comment:
Right now you have given height = 100 for every container, what if I want to occupy some space on every device? It won't be responsive. The box with height 100 will occupy less space on larger screen device and more space on a smaller screen device.

Instead of that, you can use the `Expanded` widget. The `Expanded` widget will occupy every remaining space on the screen.

Wrap your `Container` with `Expanded` widget like given below.

```dart
Expanded(
  child: Container(
    ... 
  )
)
```

Note: Whenever you use Expanded, don't give constraints to the Container. Remove the height parameter once you wrap Container with Expanded.  When you will wrap Container with Expanded, you three Containers will be joint again, to have vertical space between them, you can use `SizedBox(height: 16)`